web/src/engine/js-processor/src/jsKeyboardProcessor.ts

```diff
@@ -17,7 +17,7 @@ import { type OutputTargetBase }  from "./outputTargetBase.js";
 import { RuleBehavior }  from "./ruleBehavior.js";
 import { JSKeyboardInterface }  from './jsKeyboardInterface.js';
 import { DeviceSpec, globalObject } from "@keymanapp/web-utils";
-import { type MutableSystemStore, SystemStoreIDs } from "./systemStores.js";
+import { type MutableSystemStore, SystemStoreIDs } from "keyman/engine/keyboard";
```

Comment:
we could add this to the import of the the other types of keyman/engine/keyboard in lines 12/13.